sql-plugin/src/main/scala/com/nvidia/spark/rapids/higherOrderFunctions.scala

```diff
@@ -277,20 +265,148 @@ case class GpuArrayTransform(
     }
   }
 
+  protected def transformListColumnView(
```

Comment:
Can we have a few comments here explaining what this does and what the life cycle of `lambdaTransformedCV` is. i.e. it is required the the implementation of this close it?